client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -983,3 +983,200 @@ main.akismet-checkout-thank-you {
 .jetpack-instruction-list__item .gridicon {
 	margin-bottom: -3px;
 }
+
+// Design i2 overrides, the purchase-detail component is also used by the /plans/my-plan page to render plan features.
```

Comment:
Since we want to override it and have everything isolated, should we move this code to a new file inside `redesign-v2` folder as well?

Reply:
Moved! https://github.com/Automattic/wp-calypso/pull/78931/commits/552c7c7377dff0874bcfe01c400f6480521ab1e4